go.mod

```diff
@@ -20,7 +20,6 @@ require (
 	github.com/imdario/mergo v0.3.7
 	github.com/instrumenta/kubeval v0.0.0-20190804145309-805845b47dfc
 	github.com/justinbarrick/go-k8s-portforward v1.0.4-0.20190722134107-d79fe1b9d79d
-	github.com/modern-go/reflect2 v1.0.1 // indirect
```

Comment:
how is this ...

Reply:
I have had this for any PR I have been working on in the past days, and decided to fix it here.

Reply:
Use `go mod`, they said, it'll be so much better ...